mmv1/third_party/terraform/fwprovider/framework_provider_test.go.erb

```diff
@@ -1,13 +1,19 @@
 <% autogen_exception -%>
-package google
+package fwprovider_test
 
 import (
 	"fmt"
 	"regexp"
+	"strings"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
 	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/fwtransport"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
```

Comment:
These get rewritten right?

Reply:
Sorry, I didn't get the question. Do you mind giving more details?